package.json

```diff
@@ -68,6 +68,7 @@
     "proxy-polyfill": "0.1.6",
     "re-reselect": "^4.0.0",
     "react-beautiful-dnd": "^13.1.0",
+    "react-color": "^2.19.3",
     "react-datetime-picker": "^3.4.3",
     "react-plastic": "^1.1.1",
```

Comment:
### Slime location
- On iOS Safari when logged in, everything looks correct, but when logged out, the slime is misplaced
- Also, note how Repost and Share is somewhat overlapping

![image](https://user-images.githubusercontent.com/64950861/150709101-f219c3e0-da88-43e8-92e9-563c8bef2616.png)

Reply:
Fixed. There was a class conflict. The guest mode is using `a` tags instead of `button` to redirect the users to the login/signup page.
![image](https://user-images.githubusercontent.com/94839278/150745312-984209c9-a027-48dc-9e34-e1c99acfa876.png)